backend/dataall/modules/dataset_sharing/services/share_processors/s3_bucket_process_share.py

```diff
@@ -88,6 +90,11 @@ def process_approved_shares(
                 env_group,
             )
             try:
```

Comment:
Why do we add this check in the buckets `process_approved_shares` but we do not do it for access points or tables? I would make it fail before triggering the ECS task, like the next steps of adding a check in the approve of the share

Reply:
^^